gunicorn/arbiter.py

```diff
@@ -92,6 +92,8 @@ def setup(self, app):
         self.cfg = app.cfg
 
         if self.log is None:
+            if self.cfg.instrumentation_classes:
+                glogging.add_instrumentation(self.cfg)
```

Comment:
I'd just call `glogging.add_instrumentation(self.cfg)` instead of conditioning it on `self.cfg.instrumentation_classes` here.  If `cfg.instrumentation_classes` returns no classes, the `add_instrumentation` call will just be a no-op.